### DISCUSSION TITLE
Can't connect to any deploy host ( vercel, netlify )

### DISCUSSION THREAD

**Discussion by ErickArgollo** [community/community#35348] [Other Feature Feedback, Questions, & Ideas]
Can anyone help me?
[https://imgur.com/a/RtIuIXk](url)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ErickArgollo ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
